cert/spec/commands_generator_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'cert/commands_generator'
+
+describe Cert::CommandsGenerator do
+  let(:runner) do
+    runner = Cert::Runner.new
+    expect(Cert::Runner).to receive(:new).and_return(runner)
+    runner
+  end
+
+  describe ":create option handling" do
+    it "username short flag from tool options can be used" do
+      # leaving out the command name defaults to 'create'
+      stub_commander_runner_args(['-u', 'me@it.com'])
+
+      # launch takes no params, but we want to expect the call and prevent
+      # actual execution of the method
+      expect(runner).to receive(:launch)
+
+      Cert::CommandsGenerator.start
+
+      expect(Cert.config[:username]).to eq('me@it.com')
+    end
+
+    it "platform flag from tool options can be used" do
```

Comment:
Could we rephrase this to be something like `it "can use platform flag from tool options"` to make it read a little nicer or is that confusing the phrase?